packages/frames-client/src/constants.ts

```diff
@@ -1,3 +1,3 @@
-export const OPEN_FRAMES_PROXY_URL = "https://open-frames-proxy.fly.dev/";
+export const OPEN_FRAMES_PROXY_URL = "https://frames.xmtp.chat/";
```

Comment:
I'm now hosting this on Cloudflare Workers, which should be faster since they are closer to the end user.